docs/framework/litellm.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: "Using Composio With LiteLLM"
+sidebarTitle: "LiteLLM"
+icon: "people-group"
+description: "Integrate Composio with LiteLLM and seamlessly interact with external apps"
+---
+
+## Star A Repository on Github
+In this example, we will use LiteLLM to star a repository on Github using Composio Tools
+
+<Steps>
+<Step title="Install Packages">
+```bash Python
+ pip install composio_openai litellm
+```
+</Step>
+<Step title="Import Libraries, Initialize ComposioToolSet & Create Router">
+```bash Python
```

Comment:
Similar to the previous comment, the code block specifies ```bash Python``` which is incorrect. Since this is Python code, use ```python``` instead.